apps/mosaico/src/Article/Article.purs

```diff
@@ -141,46 +141,58 @@ render { props, state, setState } =
         mainImage = (_.mainImage =<< letteraArticle) <|> (_.listImage =<< props.articleStub)
```

Comment:
There seems to be something funny going on with the styles: It renders mobile(?) view of the front page, and any subsequent page render, for instance if you go to an article. However, if you land directly on an article it shows desktop styles https://20210929t131447-dot-mosaico-dot-ksf-staging.uc.r.appspot.com/artikel/d6d08500-ddfe-4566-9f1a-8190345831f7

Reply:
Did this relate to the Lettera problem, or is it something different?

Reply:
Seems not to, still looks weird (at least in my browser)

Reply:
Ouch. Looks like we might have merged back in from the remote branch some code we deleted. I think this is something to fix first thing at the next mob programming session.

Reply:
We noticed it gets different css classes when rendered ssr compared to client side. I thought we fixed that, but apparently not completely.